Documentation/coding-guidelines/coding-style.md

```diff
@@ -33,6 +33,8 @@ The general rule we follow is "use Visual Studio defaults".
 
 We have provided a Visual Studio 2013 vssettings file (`corefx.vssettings`) at the root of the corefx repository, enabling C# auto-formatting conforming to the above guidelines. Note that rules 7 and 8 are not covered by the vssettings, since these are not rules currently supported by VS formatting.
```

Comment:
You can also delete "2013" from this paragraph while you are on it. The vssettings file is not specific to VS 2013.